src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/CoveragePoNNormalizationResult.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.tools.pon.coverage;
+
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+
+/**
+ * Interface for a coverage profile that has been normalized by a {@link CoveragePanelOfNormals}.
+ */
+public interface CoveragePoNNormalizationResult {
+    ReadCountCollection getProfile();
```

Comment:
I think you can give this the `@FunctionalInterface` annotation, which would keep the compiler from complaining if anyone ever wanted to instantiate the interface as a lambda.